Makefile

```diff
@@ -548,103 +569,125 @@ $(TOOLCHAIN):
 	git config --global core.autocrlf false
 	mkdir -p $(TOOLCHAIN)
 
+$(PATCH_DIR):
```

Comment:
As I remember, there was this issue in pass2 only, I will check that.

Here the same in Pfalcon's:
> [INFO ]  Installing final compiler
> [ERROR]    xgcc: error: spawn: No such file or directory
> [ERROR]    /home/Peter/Pfalcon/crosstool-NG/.build/xtensa-lx106-elf/build/build-cc-final/xtensa-lx106-elf/libstdc++-v3/include/valarray:1233:1: internal compiler error: Segmentation fault
> [ERROR]    make[6]: *** [xtensa-lx106-elf/bits/stdc++.h.gch/O2g.gch] Error 1
> [ERROR]    /home/Peter/Pfalcon/crosstool-NG/.build/xtensa-lx106-elf/build/build-cc-final/xtensa-lx106-elf/libstdc++-v3/include/bits/unordered_set.h:1294:1: internal compiler error: Segmentation fault

Greetings

Am 23.12.2015 um 01:57 schrieb Dmitry Kireev:
> In Makefile
> <https://github.com/kireevco/esp-alt-sdk/pull/8#discussion_r48314895>:
>
>> @@ -548,103 +569,125 @@ $(TOOLCHAIN):
>>  	git config --global core.autocrlf false
>>  	mkdir -p $(TOOLCHAIN)
>>
>> +$(PATCH_DIR):
>
> Should not that be a part of either pass1 or pass2 of gcc?
> Also, I have other patches that should go into src/patches, so this task
> won't probably wokr for other patches (some prereqs will be missing).
> Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kireevco/esp-alt-sdk/pull/8/files#r48314895>.
>